addons/portal_rating/static/src/chatter/frontend/message_patch.xml

```diff
@@ -13,7 +13,7 @@
                 is link to the message (because publisher comment data
                 is on the rating.rating model - one comment max) -->
             <div t-if="message.rating and message.rating.id" class="o_wrating_publisher_container">
-                <button t-if="store.self.is_user_publisher"
+                <button t-if="message.effectiveSelf.is_user_publisher"
```

Comment:
This one is groups/ACL related so it should stay on `store.self` (cannot impersonate access of others even with the portal token in `/website/rating/comment`)